py/openage/pack/pack_config.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+import configparser
+import re
+
+
+class PackConfig:
+    """
+    main config file for a mod pack.
+
+    stores the package metainformation for the mod list.
+
+    file contents:
+
+    [openage-pack]
+    name = <package name>
+    version = <package version>
```

Comment:
What precisely is the package version?
Is it the version of the used package format, or the actual revision of the package content?
You might need two different, more precisely named fields here.